package.json

```diff
@@ -21,51 +21,43 @@
     "url": "https://github.com/publiclab/plots2/issues"
   },
   "dependencies": {
-    "Blob.js": "eligrey/Blob.js#*",
-    "FileSaver": "eligrey/FileSaver.js#*",
+    "blob.js": "^1.0.1",
     "blueimp-file-upload": "blueimp/jQuery-File-Upload#^9.22.1",
-    "bootstrap": "twbs/bootstrap#~3.2.0",
-    "bootstrap-3-typeahead": "bassjobsen/Bootstrap-3-Typeahead#~4.0.0",
-    "bootstrap-datepicker": "eternicode/bootstrap-datepicker#~1.6.0",
-    "bootstrap-tokenfield": "sliptree/bootstrap-tokenfield#~0.12.0",
-    "bootstrap-typeahead": "bassjobsen/Bootstrap-3-Typeahead#~4.0.0",
-    "chart.js": "chartjs/Chart.js#v2.7.0",
-    "d3": "mbostock-bower/d3-bower#~3.5",
-    "d3-bower": "mbostock-bower/d3-bower#~3.5",
-    "d3-plugins": "d3/d3-plugins#*",
-    "file-saver": "eligrey/FileSaver.js#*",
-    "flot": "flot/flot#~0.7",
-    "font-awesome": "FortAwesome/Font-Awesome#~4.5.0",
-    "fontawesome": "FortAwesome/Font-Awesome#~4.5.0",
-    "horsey": "bevacqua/horsey#3.0.0",
-    "i18n-js": "fnando/i18n-js#^2.1.2",
-    "ics": "nwcell/ics.js#0.1.3",
-    "ics.js": "nwcell/ics.js#0.1.3",
-    "image-sequencer": "publiclab/image-sequencer#^2.2.3",
-    "inline-markdown-editor": "publiclab/inline-markdown-editor#~0.3.0",
-    "jquery": "jquery/jquery-dist#~> 3.0.0",
+    "bootstrap": "^3.2.0",
+    "bootstrap-3-typeahead": "^4.0.0",
+    "bootstrap-datepicker": "^1.6.0",
+    "bootstrap-tokenfield": "^0.12.0",
+    "chart.js": "^2.7.0",
+    "cytoscape": "^3.3.2",
+    "file-saver": "^2.0.0",
+    "flot": "^0.8.3",
+    "font-awesome": "^4.5.0",
+    "horsey": "^3.0.0",
+    "i18n-js": "^3.2.1",
+    "ics": "nwcell/ics.js^0.2.0",
```

Comment:
Tried changing this to `ics` via npm but that's actually a different project https://github.com/adamgibbons/ics (maybe better maintained)

whereas we're using: https://github.com/nwcell/ics.js